github_api_helper.js

```diff
@@ -118,22 +118,50 @@ function getConclusion (annotations) {
  * @param {import('probot').Context} context Probot context
  * @param {Number} runID check run identifier
  * @param {Object} output merged scan output
+ * @param {Number} annotationsPerPage Optional: number of annotations to send with one API call.
  * @returns {Promise<any>} GitHub response
  * See: https://developer.github.com/v3/checks/runs/#update-a-check-run
  */
-function sendResults (context, runID, output) {
+function sendResults (context, runID, output, annotationsPerPage) {
   const { owner, repo } = context.repo()
-  const completedAt = new Date().toISOString()
+  let numAnnotationsLeftToSend = output.annotations.length
 
-  return context.github.checks.update({
-    check_run_id: runID,
-    owner,
-    repo,
-    status: 'completed',
-    completed_at: completedAt,
-    conclusion: getConclusion(output.annotations),
-    output
-  })
+  let numAnnotationsPerAPICall = annotationsPerPage || config.numAnnotationsPerUpdate
+  if (numAnnotationsPerAPICall <= 0 || numAnnotationsPerAPICall > 50) {
```

Comment:
50 is used several times, making it deserving of a constant

Reply:
Yep, I added a constant.